### PR TITLE
adding simple support for youtube playlists

### PR DESCRIPTION
**Pull request by drhoet**
Adding simple support for youtube playlist URLs

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -221,8 +229,13 @@ function parseUrlQueue(url, pathname, playhost) {
   return;
 }
 //Send a YouTube video
-function sendYouTube(ytid, playhost) {
-  var url = 'plugin://plugin.video.youtube/?action=play_video&videoid=' + ytid;
+function sendYouTubeVideo(ytid, playhost) {
+  var url = 'plugin://plugin.video.youtube/play/?video_id=' + ytid;
```
Comment (DuncanvR) [inline lib/main.js]: This will also address #29 :+1:

**Comment by dirkjanm** [comment]
This looks good, work on playlists is being done in the [playlists feature](https://github.com/dirkjanm/firefox-send-to-xbmc/tree/playlists-feature) branch, I'm not sure yet when that one will be finished since it requires quite some thinking about what to do with the several buttons we're getting.
Could you make your pull request to that branch so I can merge it in already?

**Comment by drhoet** [comment]
See #35 
There are however some other changes already going on that branch, e.g. the change of method name to parseUrl and the added call to  plists.add_item();
I didn't immediately see how to merge it correctly ;-)

**Comment by dirkjanm** [comment]
Closing in favor of #35

**Pull request closed without merging**